packages/tradershub/src/features/cfd/screens/MT5TradeScreen/MT5TradeScreen.tsx

```diff
@@ -0,0 +1,155 @@
+                                : PlatformDetails[platform as keyof typeof PlatformDetails].icon}
+                        </div>
+                        <div className='flex flex-col'>
+                            <div className='flex items-center flex-row gap-300'>
+                                <Text size='md'>
+                                    {platform === mt5Platform
+                                        ? MarketTypeDetails[(marketType ?? 'all') as keyof typeof MarketTypeDetails]
+                                              .title
+                                        : PlatformDetails[platform as keyof typeof PlatformDetails].title}{' '}
+                                    {!activeData?.is_virtual && details?.landing_company_short?.toUpperCase()}
+                                </Text>
+                            </div>
+                            <Text className='text-system-light-less-prominent-text' size='sm'>
+                                {loginId}
+                            </Text>
+                        </div>
+                    </div>
+                    <div className='flex flex-col items-end'>
+                        <Text bold>{details?.display_balance}</Text>
+                    </div>
+                </div>
+                <div className='flex flex-col gap-100'>
+                    {/* <div className='wallets-mt5-trade-screen__details-clipboards'> */}
+                    {platform === mt5Platform && (
+                        <Fragment>
+                            <MT5TradeDetailsItem label='Broker' value='Deriv Holdings (Guernsey) Ltd' />
+                            <MT5TradeDetailsItem
+                                label='Server'
+                                value={(details as THooks.MT5AccountsList)?.server_info?.environment ?? 'Deriv-Server'}
+                            />
+                            <MT5TradeDetailsItem label='Login ID' value={loginId ?? '12345678'} />
+                            <MT5TradeDetailsItem label='Password' value='********' variant='password' />
+                        </Fragment>
+                    )}
+                    {platform === dxtradePlatform && (
+                        <Fragment>
+                            <MT5TradeDetailsItem
+                                label='Username'
+                                value={(details as THooks.DxtradeAccountsList)?.login ?? '12345678'}
+                            />
+                            <MT5TradeDetailsItem label='Password' value='********' variant='password' />
+                        </Fragment>
+                    )}
+                    {platform === ctraderPlatform && (
+                        <MT5TradeDetailsItem
+                            value=' Use your Deriv account email and password to login into the cTrader platform.'
+                            variant='info'
+                        />
+                    )}
+                </div>
```

Comment:
The name suggests as it is a MT%TradeScreen
SO shouldn't it contains only MT5 data.
There should be TradeScreen or vice versa if we are reusing the screen for 
Ctrader and Dxtrade

Reply:
@hamza-deriv fully agree with you, but in wallets packages it named like this and @shontzu-deriv already created MT5TradeDetailsItem and MT5TradeLink that was used for all types of trades in wallets and I made decision didn't rename it and follow wallets name way and rename it later to prevent some conflicts right now. But I can try to rename it right now, let me know

Reply:
I think we need to separate it out otherwise it will be like this.
@mahdiyeh-deriv need your input on this. It should not be gone forward.
But as this PR is already a big one, I suggest a separate card to break this down into smaller files.

Reply:
I've added this card for the improvement mentioned above and assigned it to you @sergei-deriv 
cc: @hamza-deriv